package.json

```diff
@@ -31,6 +31,10 @@
   "scripts": {
     "build": "tsc",
     "clean": "del ./dist/*",
+    "fix:package.json": "run-s fix:package.json:prettier fix:package.json:sort",
```

Comment:
Because the name is long, input pain will not be eliminated.
Let's rename `fix:package.json` as `fix-pkg`.

Reply:
I think that `fmt` (abbreviation of "format") is more appropriate than `fix`.